src/network/networkd-link.c

```diff
@@ -1325,6 +1325,11 @@ static int link_set_bridge(Link *link) {
                 if (r < 0)
                         return log_link_error_errno(link, r, "Could not append IFLA_BRPORT_COST attribute: %m");
         }
+        if (link->network->priority != 0) {
```

Comment:
@ssahani that is Bridge's priority. I'm working on Bridge Port's priority which is 32 with `$ ip -d link show eth0` in the line prefixed "bridge_slave". Please see updated testsuite. Note, previously systemd would use [BridgePort] section in the .network file to define bridge port settings, but that was renamed to just [Bridge]. The sysfs file for it is `/sys/class/net/port1/brport/priority` and 128 is an out of range value for it.